Example/PaymentSheet Example/PaymentSheetUITest/LinkPaymentControllerUITest.swift

```diff
@@ -91,7 +91,7 @@ class LinkPaymentControllerUITest: XCTestCase {
             .withOffset(CGVector(dx: 0, dy: -130))
             .tap()
 
-        sleep(1) // wait for modal to disappear before pressing Buy
+        sleep(3) // wait for modal to disappear before pressing Buy
```

Comment:
What happens if you tap the buy button immediately?  This sleep is long enough to make me wonder if it's possible for customers to run into whatever issue we fixed by sleeping.